playbooks/cluster/openshift/README.md

```diff
@@ -8,4 +8,4 @@ Create an OpenShift Cluster
 | kubevirt_openshift_version| 3.7| <ul><li>3.7</li><li>3.9</li></ul>|OpenShift cluster version.|
 | openshift_ansible_dir | openshift-ansible | |Path to the openshift-ansible directory.|
 | openshift_playbook_path | playbooks/byo/config.yml |<ul><li>playbooks/byo/config.yml</li><li>playbooks/deploy_cluster.yml</li></ul>|Path to the OpenShift deploy playbook. <br>3.7: **playbooks/byo/config.yml**<br>3.9: **playbooks/deploy_cluster.yml**|
-| storage_role | | <ul><li>storage-glusterfs</li></ul> | Storage flavor to deploy in the cluster. Don't forget to add gluster nodes in the [inventory file](https://github.com/kubevirt/kubevirt-ansible/blob/master/inventory).|
+| storage_role | | <ul><li>storage-glusterfs</li></ul> | Storage flavor to deploy in the cluster. Don't forget to add gluster nodes in the [inventory file](https://github.com/kubevirt/kubevirt-ansible/blob/master/inventory). You will also need to add a DNS record that maps between heketi-{{ glusterfs_name }}.{{ openshift_master_default_subdomain }}" to the IP of the node which runs Openshift's router.|
```

Comment:
Try something like this instead:
``For OpenShift, you will also need to have DNS configured such that `heketi-{{ glusterfs_name }}-{{ glusterfs_namespace }}.{{ openshift_master_default_subdomain }}` resolves to the IP address of a node running an OpenShift router.``